var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -46,3 +80,9 @@ class PyAwkward(PythonPackage):
     @when("@1.9.0:")
     def setup_build_environment(self, env):
         env.set("CMAKE_ARGS", "-DAWKWARD_EXTERNAL_PYBIND11=TRUE")
+
+    @run_after("install")
+    @on_package_attributes(run_tests=True)
+    def install_test(self):
```

Comment:
Probably shouldn't assume the testing works for pre-v2.
```suggestion
    @when("@2:")
    @run_after("install")
    @on_package_attributes(run_tests=True)
    def install_test(self):
```

Reply:
Just make sure these decorators actually stack, I've had issues with this in the past.

Reply:
I also wouldn't assume that the tests don't work. You also don't have to add tests if you don't want to, I never do.

Reply:
This set of decorators indeed doesn't stack. I don't know if decorator order matters. If it doesn't, then I guess we can just drop the tests and the whole set of test dependencies.

Reply:
Order does matter, @alalazo explained why once

Reply:
The following is at least more likely to succeed (still no precedent for it):
```suggestion
    @run_after("install", when="@2:")
    @on_package_attributes(run_tests=True)
    def install_test(self):
```
but I'm just removing the tests.